Gemfile

```diff
@@ -29,6 +29,7 @@ gem 'bootsnap', '>= 1.4.2', require: false
 gem 'bootstrap', '~> 4.4.1'
 gem 'haml-rails', '>= 2.0', :group => :development
 gem 'jquery-rails'
+gem 'pg', '~> 1.2.3'
```

Comment:
Since you'll be using Postgresql instead of sqlite, the sqlite3 gem is no longer required. So feel free to get rid of these [lines](https://github.com/ValeriaMejiaCortes/todo-app-rails/pull/11/files#diff-d09ea66f8227784ff4393d88a19836f321c915ae10031d16c93d67e6283ab55fR8-R9) and then make sure to run `bundle install`.

Reply:
Please address this comment before marking it as resolved

Reply:
@ValeriaMejiaCortes please address this requested change